src/components/timeline/TrackContextMenu.js

```diff
@@ -407,9 +411,20 @@ class TimelineTrackContextMenuImpl extends PureComponent<Props> {
       // Is there only one visible global track?
       globalTracks.length - hiddenGlobalTracks.size === 1;
 
+    const rightClickedTrackName = this.getRightClickedTrackName(
+      rightClickedTrack
+    );
+
     return (
       <MenuItem onClick={this._isolateProcessMainThread} disabled={isDisabled}>
-        Only show {`"${this.getRightClickedTrackName(rightClickedTrack)}"`}
+        <Localized
+          id="TrackContextMenu--only-show-track"
+          vars={{ trackName: rightClickedTrackName }}
+        >
+          <>
+            Only show {`“${this.getRightClickedTrackName(rightClickedTrack)}”`}
```

Comment:
nit: please use `rightClickedTrackName` directly, or remove the full string keeping `<Localized/>` without fallback text?
Is `<> </>` necessary because we use a var? I think it's necessary only for `elems`, can you please double check?

Reply:
Same for all the cases below

Reply:
Ah, I see, I think that's 2 text nodes...
It's probably not needed if we keep it empty :) but up to you

Reply:
I feel like we are going to change all the Localized components soon anyway and remove the fallback strings. This should be short-lived :)